duckduckgo/macos-browser
# Bump privacy dashboard to latest version

Task/Issue URL: https://app.asana.com/0/1177771139624306/1200473629312157/f
Tech Design URL:
CC: @tomasstrba @mallexxx @samsymons

**Description**:
Updates Privacy Dashboard
- https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/23
- https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/25 (cc: @mallexxx)

**Steps to test this PR**:
1. Go to macOS System Prefernces > General and change the "Appearance" to "Dark"
1. Privacy Dashboard should now appear with dark theme

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
